src/main/ApplicationImpl.cpp

```diff
@@ -1306,6 +1306,9 @@ ApplicationImpl::syncOwnMetrics()
     }
     mMetrics->NewCounter({"process", "file", "handles"})
         .set_count(fs::getOpenHandleCount());
+
+    // Update simple timer metrics
+    mMetrics->syncMaxes();
```

Comment:
I think this would be a good place for a bit of a longer comment explaining why we need to periodically sync the SimpleTimer.